### ISSUE TITLE
GitFlow button not appearing

### ISSUE THREAD

**Issue by pulvegrip** [jakobehn/GitFlow.VS#93]
Hi,

the code on master doesn't build on a VS2019 machine without some changes to references and a rather strange manual copy of a manifest file to the obj/Debug folder, this is an inconvenience not an issue really.

The issue I would like to report is that once the code builds and the extension is installed it does not add itself to the Team Explorer page at all, when debugging we can see that the extension is in some fashion loaded since Constructor and the InitializeAsync, GetService,  methods of the GitFlowVSExtension class are run.

The extension that can be downloaded from Visual Studio Marketplace is at version 1.3.15 and the one here is 1.3.12.

What am I missing here?

**Comment by melyreturns** [comment]
Hello,

This is still happening, is there a temporary solution I can fix this?

**Comment by jakobehn** [comment]
Hi, when building and running the solution locally, you should use the VS2019 configuration. Let me know if that doesn't work

**Comment by melyreturns** [comment]
Sorry, it's not working for me. I'm getting this error:

![image](https://user-images.githubusercontent.com/38630271/76874606-2d010200-684e-11ea-9163-afedd681f969.png)

I checked if the extension is enabled too:

![image](https://user-images.githubusercontent.com/38630271/76874918-97b23d80-684e-11ea-9d1a-847eb45c4ce2.png)

I'm missing something else? This issue started when disabled auto updates on vs2019, but I enabled these options again.
